src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
+* {
```

Comment:
Don't use * selector (it impacts performance)
Better use ``html`` or ``body`` tag.
![image](https://github.com/mate-academy/layout_product-cards/assets/54860641/24e6760a-51e7-477a-9c91-0e0ff81063a7)